src/Chapter1.hs

```diff
@@ -477,8 +478,8 @@ Implement a function that returns the last digit of a given number.
   results. Or you can try to guess the function name, search for it and check
   whether it works for you!
 -}
--- DON'T FORGET TO SPECIFY THE TYPE IN HERE
-lastDigit n = error "lastDigit: Not implemented!"
+lastDigit :: Integral n => n -> n
+lastDigit = (`mod` 10) . abs
```

Comment:
Nice point free style 💯

Reply:
Yes, that's elegant. However, to me the alternative with explicit arguments is sometimes easier to grasp than the point free one, especially when infix notation is used.

I guess to a certain degree it will get better if I get used to it, but probably it makes sense to not use point free style sometimes